docs/modules/Phalcon.md

```diff
@@ -26,7 +26,7 @@ Please try it and leave your feedback.
 ## Config
 
 The following configurations are required for this module:
-* boostrap: the path of the application bootstrap file</li>
+* bootstrap: the path of the application bootstrap file</li>
```

Comment:
There needs to be a blank line before this for Markdown to recognise this as a list. Also there is HTML at the end of the line.